src/pgo.erl

```diff
@@ -57,14 +58,14 @@ query(Query) ->
 query(Conn=#conn{}, Query) ->
     pgo_handler:simple_query(Conn, Query);
 query(Pool, Query) when is_atom(Pool) ->
-    %% SpanCtx = oc_trace:start_span(<<"pgo:query/2">>, ocp:current_span_ctx(),
-    %%                               #{attributes => #{<<"query">> => Query}}),
+    SpanCtx = oc_trace:start_span(<<"pgo:query/2">>, ocp:current_span_ctx(),
+                                  #{attributes => #{<<"query">> => Query}}),
```

Comment:
Any reason not to use the Erlang built-in macros for module, function name and arity? I’m fine without it.

Reply:
Eh, just because it would have to still be constructed at runtime if doing that.